litmus-portal/backend/subscriber/pkg/gql/subscriptions.go

```diff
@@ -55,8 +55,7 @@ func ClusterConnect(clusterData map[string]string) {
 	for {
 		_, message, err := c.ReadMessage()
 		if err != nil {
-			log.Println("read:", err)
-			return
+			log.Fatal("SUBSCRIPTION ERROR : ", err)
```

Comment:
Why we are removing return

Reply:
The subscriber needs to halt and restart, to reconnect to the subscription.